examples/custom-animation/running-dashed-message-flow/index.js

```diff
@@ -0,0 +1,14 @@
+// Main BPMN Container
```

Comment:
We only have a single bpmn-container (this seems taken from the call activity custom navigation example
```suggestion
```